### PR TITLE
Fix threatened dreams and killing in the name of storage overlap

### PR DESCRIPTION
**Pull request by Deadsmelly**
# Description

Poacher chest for Threatened Dreams gives a storage ID assigned to the Killing In The Name Of Hellhound boss (34123)

### **Actual**

Doing the Threatened Dreams quest allows you to kill the Hellhound boss because it gives the boss's storageID.

### **Expected**

Not be able to kill hellhound boss by doing Threatened Dream Quest.

## Fixes

N/A

## Type of change

Please delete options that are not relevant.

  - [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested

Changed values, was able to do quest with new values, no storage overlap.

**Test Configuration**:

  - Server Version: Latest
  - Client: N/A
  - Operating System: N/A

## Checklist

  - [X] My code follows the style guidelines of this project
  - [X] I have performed a self-review of my own code
  - [X] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [X] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by majestyotbr — APPROVED** [review]

File: data/startup/tables/chest.lua [hunk 1]
```diff
@@ -1001,7 +1001,7 @@ ChestUnique = {
 	},
 	-- Others uniques
 	-- Threatened Dreams Quest
-	[34123] = {
+	[50306] = {
```
Comment (dudantas) [inline data/startup/tables/chest.lua]: What is the reason for this change? The table was made to have pre-defined unique and action ranges, so as not to risk repeating. 

#Edit
Ok, I checked here and someone added this random unique there. Please change to 14036?

Comment (Deadsmelly) [reply data/startup/tables/chest.lua]: Changed

**Review by majestyotbr — APPROVED** [review]
